newIDE/app/src/MainFrame/EditorContainers/StartPage/Carousel.js

```diff
@@ -0,0 +1,338 @@
+// @flow
+import * as React from 'react';
+import GridList from '@material-ui/core/GridList';
+import { GridListTile, Paper } from '@material-ui/core';
+import ArrowBackIos from '@material-ui/icons/ArrowBackIos';
+import ArrowForwardIos from '@material-ui/icons/ArrowForwardIos';
+import { Skeleton } from '@material-ui/lab';
+import Text from '../../../UI/Text';
+import './Carousel.css';
+import { Column, Line, Spacer } from '../../../UI/Grid';
+import GDevelopThemeContext from '../../../UI/Theme/ThemeContext';
+import { useResponsiveWindowWidth } from '../../../UI/Reponsive/ResponsiveWindowMeasurer';
+
+export type YoutubeThumbnail = {|
+  link: string,
+|};
+
+export type ShowcaseThumbnail = {|
+  title: string,
+  imageSource: string,
+|};
+
+type GameTemplateThumbnail = {|
+  title: string,
+  imageSource: string,
+|};
+
+type SkeletonThumbnail = {|
+  skeleton: boolean,
+  key: string,
+|};
+
+type Props = {|
+  title: string,
```

Comment:
Consider passing a React.Node so that the parents can pass a `title={<Trans>Showcase</Trans>}`